python/python/glide/async_commands/core.py

```diff
@@ -2746,6 +2746,89 @@ async def zdiffstore(self, destination: str, keys: List[str]) -> int:
             ),
         )
 
+    async def zrandmember(self, key: str) -> Optional[str]:
+        """
+        Returns a random element from the sorted set stored at 'key'.
+
+        See https://valkey.io/commands/zrandmember for more details.
+
+        Args:
+            key (str): The key of the sorted set.
+
+        Returns:
+            Optional[str]: A string representing a random element from the sorted set.
+                If the sorted set does not exist or is empty, the response will be None.
+
+        Examples:
+            >>> await client.zrandmember("my_sorted_set")
+                "GLIDE"  # "GLIDE" is a random member of "my_sorted_set".
+            >>> await client.zrandmember("non_existing_sorted_set")
+                None  # "non_existing_sorted_set" is not an existing key, so None was returned.
+        """
+        return cast(
+            Optional[str],
+            await self._execute_command(RequestType.ZRandMember, [key]),
+        )
+
+    async def zrandmember_count(self, key: str, count: int) -> List[str]:
+        """
+        Retrieves random elements from the sorted set stored at 'key'.
+
+        See https://valkey.io/commands/zrandmember for more details.
+
+        Args:
+            key (str): The key of the sorted set.
+            count (int): The number of elements to return.
+                If `count` is positive, returns unique elements.
+                If negative, allows for duplicates.
+
+        Returns:
+            List[str]: A list of elements from the sorted set.
+                If the sorted set does not exist or is empty, the response will be an empty list.
+
+        Examples:
+            >>> await client.zrandmember("my_sorted_set", -3)
+                ["GLIDE", "GLIDE", "PYTHON"]  # "GLIDE" and "PYTHON" are random members of "my_sorted_set".
+            >>> await client.zrandmember("non_existing_sorted_set", 3)
```

Comment:
we should remove this example

Reply:
See my comment [above](https://github.com/Bit-Quill/glide-for-redis/pull/276#discussion_r1600345031)